core/core.go

```diff
@@ -233,6 +265,86 @@ func (app *App) periodicallyCheckForNewAddrs(startingAddrs []ma.Multiaddr) {
 	}
 }
 
+// ErrSnapshotNotFound is the error returned when a snapshot not found with a particular id
+type ErrSnapshotNotFound struct {
+	id string
+}
+
+func (e ErrSnapshotNotFound) Error() string {
+	return fmt.Sprintf("No snapshot found with id: %s. To create a new snapshot, send a request with an empty snapshotID", e.id)
+}
+
+// GetOrders retrieves paginated orders from the Mesh DB at a specific snapshot in time. Passing an empty
+// string as `snapshotID` creates a new snapshot and returns the first set of results. To fetch all orders,
+// continue to make requests supplying the `snapshotID` returned from the first request.
```

Comment:
Maybe we should add to this doc comment the fact the snapshots expire after 1 minute of receiving no requests for that snapshot?